tools/rtpsrelay/RelayHandler.cpp

```diff
@@ -278,9 +278,8 @@ void VerticalHandler::process_message(const ACE_INET_Addr& remote_address,
     OpenDDS::DCPS::RepoId src_guid;
     GuidSet to;
 
-    OpenDDS::RTPS::MessageParser mp(*msg);
-    if (!parse_message(mp, msg, src_guid, to, true)) {
-      ACE_ERROR((LM_WARNING, ACE_TEXT("(%P|%t) %N:%l WARNING: VerticalHandler::process_message %C failed to parse_message from %C\n"), name_.c_str(), addr_to_string(remote_address).c_str()));
+    if (!parse_message(mp, msg, src_guid, to, true, now)) {
```

Comment:
Great. Thanks, github.
```suggestion
    OpenDDS::RTPS::MessageParser mp(*msg);
    if (!parse_message(mp, msg, src_guid, to, true, now)) {
```